OSDOCS:13467: Creating new tutorial book and updating CLI tutorial (#90215)
openshift/openshift-docs: modules/getting-started-cli-granting-permissions.adoc

[id="getting-started-cli-granting-permissions_{context}"]
= Granting view permissions

> {product-title} automatically creates several service accounts in every project. The `default` service account takes responsibility for running the pods. {product-title} uses and injects this service account into every pod that launches.

Reviewer: ```suggestion
{product-title} automatically creates several service accounts in every project. The `default` service account runs the pods. {product-title} uses and injects this service account into every pod that launches.
```

Author: I'm not positive if this changes the meaning, so I think I'll leave it as is.

By default, the `default` service account has limited permissions to interact with the OpenShift API.

> This procedure assigns the `view` role to the `default` service account to allow it to communicate with the OpenShift API to learn about pods, services, and resources within the project.

Reviewer 2: Maybe something like:

As a requirement of this application, this procedure assigns the `view` role to the `default` service account to allow it to communicate with the OpenShift API to learn about pods, services, and resources within the project.

> As a requirement of the application, this procedure assigns the `view` role to the `default` service account to allow it to communicate with the OpenShift API to learn about pods, services, and resources within the project.

Reviewer: ```suggestion
As a requirement of the application, you need to assign the `view` role to the `default` service account to allow it to communicate with the OpenShift API to learn about pods, services, and resources within the project.
```
